src/components/graph/selectionToolbox/BypassButton.spec.ts

```diff
@@ -0,0 +1,114 @@
+import { mount } from '@vue/test-utils'
+import { createPinia, setActivePinia } from 'pinia'
+import PrimeVue from 'primevue/config'
+import Tooltip from 'primevue/tooltip'
+import { beforeEach, describe, expect, it, vi } from 'vitest'
+import { createI18n } from 'vue-i18n'
+
+import BypassButton from '@/components/graph/selectionToolbox/BypassButton.vue'
+import { Positionable } from '@/lib/litegraph/src/interfaces'
+import { LGraphEventMode } from '@/lib/litegraph/src/litegraph'
+import { useCommandStore } from '@/stores/commandStore'
+import { useCanvasStore } from '@/stores/graphStore'
+
+class MockLGraphNode implements Positionable {
+  type = 'TestNode'
+  title = 'Test Node'
+  mode = LGraphEventMode.ALWAYS
+  id = 'node-1'
+  pos = new Float64Array([26, 186])
+  move = vi.fn()
+  snapToGrid = vi.fn()
+  boundingRect: [number, number, number, number] = [0, 0, 100, 100]
+}
+
+vi.mock('@/utils/litegraphUtil', () => ({
+  isLGraphNode: vi.fn(() => true)
+}))
+
+describe('BypassButton', () => {
+  const i18n = createI18n({
+    legacy: false,
+    locale: 'en',
+    messages: {
+      en: {
+        selectionToolbox: {
+          bypassButton: {
+            tooltip: 'Bypass/Unbypass Selected Nodes'
+          }
+        },
+        'commands.Comfy_Canvas_ToggleSelectedNodes_Bypass.label':
+          'Toggle Bypass Selected Nodes'
+      }
+    }
+  })
+
+  beforeEach(() => {
+    setActivePinia(createPinia())
+    // canvasStore = useCanvasStore()
+    // commandStore = useCommandStore()
+    // mockLGraphNode = new MockLGraphNode()
+
+    vi.clearAllMocks()
+  })
+
+  const mountComponent = () => {
+    return mount(BypassButton, {
+      global: {
+        plugins: [i18n, PrimeVue],
+        directives: { tooltip: Tooltip },
+        stubs: {
+          'i-lucide:ban': true
+        }
+      }
+    })
+  }
+
+  it('should render bypass button', () => {
+    const canvasStore = useCanvasStore()
+    const mockLGraphNode = new MockLGraphNode()
+
+    canvasStore.selectedItems = [mockLGraphNode]
```

Comment:
If you mock the store import, you can stub out the return for the standard case to elide it in most of the tests.